hello-world/entrypoint.py

```diff
@@ -0,0 +1,32 @@
+#  Copyright (c) University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from pyspark.context import SparkContext
+from pyspark.sql.session import SparkSession
+
+from src.transform import example_transform
+
+
+sc = SparkContext.getOrCreate()
```

Comment:
- The latest documentation advises that we just use `SparkSession.builder.getOrCreate()`. `SparkContext` will be deprecated.
- I am keen on not having global variables if we can help it and instead directly passing `spark_context` to transform functions - like I do in Data-Forge. The reasons are the usual ones for minimising global variable use. It also does not allow me to unit test easily as I cannot easily create my own `SparkSession` with a custom config. However, this is more of a style thing so I won't object to your choice.